SOOP/SOOP_TMV/dest_path.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+from ship_callsign import ship_callsign_list
+
+
+def ships():
+    return ship_callsign_list()
+
+
+def get_platform_code(file, mode):
+    """
+    # get code from netcdf file name
+    """
+    return get_file_parts(file, mode)[4]
+
+
+def get_file_parts(file, mode):
```

Comment:
Looks like `mode` is not used in this function, so you don't need it as an argument. Same for `get_platform_code` above.